tests/conftest.py

```diff
@@ -53,6 +53,11 @@ def secret_factory_class(request):
     return secret_factory_fixture(request)
 
 
+@pytest.fixture(scope='session')
+def secret_factory_session(request):
+    return secret_factory_fixture(request)
```

Comment:
@fbalak i think we should not use the secret_factory fixture anymore. the helper function (create_secret) uses admin_key and id which are no longer suggested for both CephFS and RBD.

Its ok to still have a CBP facotry session, but I dont think we need a secret factory session. We should always use the secret which is created by default or create secrets which do not use admin ID and key.

Reply:
Ok, I added this factory session fixture for the sake of consistency with the rest of the code.